File: Serene/Serene.Script/Imports/FormContexts/Meeting.MeetingAttendeeForm.cs

namespace Serene.Meeting
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Collections;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.Runtime.CompilerServices;
    using Serene.Northwind;

    public partial class MeetingAttendeeForm : PrefixedContext
    {
        public MeetingAttendeeForm(string idPrefix) : base(idPrefix) {}
    
        public IntegerEditor MeetingId { get { return ById<IntegerEditor>("MeetingId"); } }
        public IntegerEditor ContactId { get { return ById<IntegerEditor>("ContactId"); } }
        public IntegerEditor AttendeeTypeId { get { return ById<IntegerEditor>("AttendeeTypeId"); } }
        public IntegerEditor AttendanceStatusId { get { return ById<IntegerEditor>("AttendanceStatusId"); } }
        public StringEditor IsActive { get { return ById<StringEditor>("IsActive"); } }
    }
}

File: Serene/Serene.Web/Modules/Common/Imports/ServerTypings/Meeting.MeetingAttendeeForm.ts
namespace Serene.Meeting {
    export class MeetingAttendeeForm extends Serenity.PrefixedContext {
        static formKey = 'Meeting.MeetingAttendee';

    }

    export interface MeetingAttendeeForm {
        MeetingId: Serenity.IntegerEditor;
        ContactId: Serenity.IntegerEditor;
        AttendeeType: Serenity.EnumEditor;
        AttendanceStatus: Serenity.EnumEditor;
    }

    [['MeetingId', () => Serenity.IntegerEditor], ['ContactId', () => Serenity.IntegerEditor], ['AttendeeType', () => Serenity.EnumEditor], ['AttendanceStatus', () => Serenity.EnumEditor]].forEach(x => Object.defineProperty(MeetingAttendeeForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

